supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -254,7 +255,7 @@ object TesTask {
     configIdentity.map(_.value).orElse(workflowOptionsIdentity.map(_.value))
   }
   def makeResources(runtimeAttributes: TesRuntimeAttributes,
-                    workflowExecutionId: Option[String]): Resources = {
+                    workflowExecutionId: Option[String], additionalBackendParams: Map[String, Option[String]]): Resources = {
```

Comment:
I appreciate the effort at generality here, but this new option feels similar to `workflowExecutionId` to me. I'd suggest either making it a named input like that, or adding the `workflowExecutionId` to the new param map.